nym-vpn-core/crates/nym-vpnd/src/service/vpn_service.rs

```diff
@@ -948,15 +948,15 @@ where
         &self,
         locale: String,
     ) -> Result<ParsedAccountLinks, AccountError> {
-        let account = self.load_account().await?;
-        let account_id = account.id();
+        let account = self.load_account().await.ok();
```

Comment:
nit: if account failed to get loaded, it might be useful to log the error in case it's something different from it missing. (maybe even match on that error to make sure its nothing critical)

Reply:
So this is an interesting one. Basically it's not an error if the account isn't there, it just means it was called when we are not logged in yet. I toyed with the idea to have separate API for the two use-cases, but that would mean yet another breaking change so I left it as is, at least for now